components/TextControls/examples/MDCTextControlTextFieldTypicalUseExample.m

```diff
@@ -146,6 +153,204 @@ - (void)viewWillLayoutSubviews {
 
 @end
 
+/** This view controller manages a child view controller that contains the actual example content.
+ * It overrides the child view controller's trait collection based off the user's behavior. */
```

Comment:
What a cool example harness! This feels like a great thing to pull out to a shared library.

Reply:
Yeah the `setOverrideTraitCollection:forChildViewController:` is a very useful API! I will improve this stuff in subsequent PRs and there is definite potential for reuse elsewhere